src/main/java/edu/tamu/scholars/middleware/service/SparqlHttpTriplestore.java

```diff
@@ -0,0 +1,39 @@
+package edu.tamu.scholars.middleware.service;
+
+import edu.tamu.scholars.middleware.config.model.TriplestoreConfig;
+import org.apache.jena.query.QueryExecution;
+import org.apache.jena.query.QueryExecutionFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.time.Instant;
+
+public class SparqlHttpTriplestore implements Triplestore {
+
+    private final static Logger logger = LoggerFactory.getLogger(SparqlHttpTriplestore.class);
+
+    private final TriplestoreConfig config;
+
+    public SparqlHttpTriplestore(TriplestoreConfig config) {
+        this.config = config;
+    }
+
+    @Override
+    public QueryExecution createQueryExecution(String query) {
+        return QueryExecutionFactory.sparqlService(config.getDatasourceUrl(), query);
+    }
+
+    @Override
+    public void init() {
+        Instant start = Instant.now();
+        logger.info(String.format("Initializing %s", config.getType().getSimpleName()));
```

Comment:
I imagine throughout the code base logging has been done using `String.format`. However, can we start using the logger to template variables.

```
logger.info("Initializing {}", config.getType().getSimpleName());
```